src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 40 * days;
+
+  if (days >= 7) {
+    sum -= 50;
```

Comment:
1) Consider using a constant for the discount value 50 instead of using the hardcoded value.
2) you can just use `early return`, like `return rentalCost - proDiscount`